Gruppe.A1.End-to-End_Encryption_mit_IPv6_und_integriertem_IPSec.ThushjandanPonnudurai.PascalBaumann.Termpaper.tex

```diff
@@ -215,7 +215,7 @@ \section{VPN protocol suites}
 \subsection{SSL VPN}
 \label{ssec:sslvpn}
 
-SSL VPN is a transport layer VPN protocol. The encryption and connection establishment is done over the TLS protocol. TLS (Transport Layer Security) is the successor of the SSL protocol, and thus a part of the SSL protocol suite which is widely used in the web environment. Due to this widespread adaptation, implementing an SSL VPN in an enterprise environment is relatively trivial. As TLS is reliant on a reliable transport channel, its typically implemented over TCP. \parencite[6,96]{Dierks2008}
```

Comment:
https://www.globalsign.com/en/ssl-information-center/what-is-an-ssl-certificate/
Because english is weird: spoken "an esesl" :D